ros/src/waypoint_updater/jmt.py

```diff
@@ -122,8 +122,9 @@ def set_VAJt(self, V, A, J, time):
         self.time = time
 
     def __repr__(self):
-        return "%5.3f, %2.4f, %2.4f, %2.4f, %2.3f" % (self.S, self.V, self.A,
-                                                      self.J, self.time)
+        return "%5.3f, %2.4f, %2.4f, %2.4f, %2.3f" % (self.time, self.S,
```

Comment:
May also need to reorder the float formats:
"%2.3f, %5.3f, %2.4f, %2.4f, %2.4f"